### PR TITLE
Motor doc: fix wrong comment

### PR DESCRIPTION
**Pull request by vrubiolo**
This commit also adds a oneliner stop() example.

### CODE REVIEW

**Comment by dtex** [comment]
@vrubiolo Thanks for this. Two notes:

1) The /docs files are generated from the /eg files so your change should be made in /eg and then run ```grunt examples``` to generate the updated docs/motor.md file

2) We already have a motor.stop() call in there after a two second delay. Calling stop() immediately after start() will make it appear as though nothing has happened.

**Comment by vrubiolo** [comment]
Hi Donovan,

Thanks for the feedback and the advice. I'll rework the PR as you advised. It was not obvious to me which files to modify so I might submit a patch for that as well.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ Docs should have tested and working sample code, [fritzing diagrams](http://frit
 
 The [wiki](https://github.com/rwaldron/johnny-five/wiki) contains documentation about the api, and contains code examples and fritzing diagrams.
 
-The [eg folder](https://github.com/rwaldron/johnny-five/tree/master/eg) contains working code examples that users can run out of the box. Each of these example files has an associated .md file in the [docs folder](https://github.com/rwaldron/johnny-five/tree/master/docs) that is generated from the eg files.
+The [eg folder](https://github.com/rwaldron/johnny-five/tree/master/eg) contains working code examples that users can run out of the box. Each of these example files has an associated .md file in the [docs folder](https://github.com/rwaldron/johnny-five/tree/master/docs) that is generated from the eg files. When contributing documentation updates for those code examples, modify the one in the [eg folder](https://github.com/rwaldron/johnny-five/tree/master/eg) and run 'grunt examples' to regenerate its [docs folder](https://github.com/rwaldron/johnny-five/tree/master/docs) counterpart, then submit both of them as part of your contribution/PR.
```
Comment (rwaldron) [inline CONTRIBUTING.md]: I guess this was way out of date, it needs to mention adding an entry to `tpl/programs.json` as well.

**Comment by vrubiolo** [comment]
@rwaldron: thanks ! How about this one? I also updated the grunt commandline to be in line with the other grunt references in the file

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ Docs should have tested and working sample code, [fritzing diagrams](http://frit
...
 
-The [eg folder](https://github.com/rwaldron/johnny-five/tree/master/eg) contains working code examples that users can run out of the box. Each of these example files has an associated .md file in the [docs folder](https://github.com/rwaldron/johnny-five/tree/master/docs) that is generated from the eg files.
+The [eg folder](https://github.com/rwaldron/johnny-five/tree/master/eg) contains working code examples that users can run out of the box. Each of these example files has an associated .md file in the [docs folder](https://github.com/rwaldron/johnny-five/tree/master/docs) that is generated from the eg files. When contributing documentation updates for those code examples, modify the one in the [eg folder](https://github.com/rwaldron/johnny-five/tree/master/eg) and run `grunt examples` to regenerate its [docs folder](https://github.com/rwaldron/johnny-five/tree/master/docs) counterpart, then submit both of them as part of your contribution/PR. If you add a new documentation file, remember you will need to add it to `tpl/program.json` as well.
```
Comment (rwaldron) [inline CONTRIBUTING.md]: :+1: Thanks!

**Pull request merged**